.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -14,6 +14,13 @@ https://github.com/microsoft/python-language-server/blob/master/TROUBLESHOOTING.
 -->
 
 ## Environment data
+<!--
+To find your version: 
+* (CTRL + Shift + P >> Python: Create Terminal >> Select "Output" >> Select "Python" from the dropdown menu in the top right of the output window). Look for the line Microsoft Python Language Server version X in the console.
```

Comment:
I'm actually not a fan of the instructions as they are written in the troubleshooting, so I'd like to take the opportunity to make them correct:

> Select "View: Toggle Output" from the command palette (Ctrl+Shift+P), then select "Python" in the dropdown on the right ("Python Language Server" if running Insiders).

I'd be good to update the troubleshooting doc in this PR as well.

Reply:
Ok I'm on it